.github/workflows/ci.yaml

```diff
@@ -345,7 +345,7 @@ jobs:
     needs: prepare
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.8]
```

Comment:
We only run on the version which we use inside our container, so we don't need check multible version

Reply:
Agree, however, the tests don't pass on Python 3.8.

I don't know or understand why at this point.

Reply:
ok